docs/writing-policies/metadata.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_label: "Policy metadata"
+title: "Policy metadata"
+---
+
+# Policy metadata
+
+The Kubewarden metadata file, `metadata.yaml`, serves as a vital configuration
+file that contains important information and settings related to the policies
+used within Kubewarden. This documentation aims to provide an overview of the
```

Comment:
This page is great! I think we should have added it ages ago!

On the review of it, I would reduce the mentions to providing an overview and example, plus I would mention that the metadata.yaml gets used by `kwctl scaffold`. Something like

> The policy `metadata.yaml` file contains defaults for the policy, in addition to metadata such as author and description, set by the policy author.
>
> When policy users want to use a policy, they generate a YAML manifest using `kwctl scaffold`. `kwctl scaffold` reads the `metadata.yaml` of the policy, performs sanity checks, and returns a YAML manifest that the author can use as-is or adapt to their needs.
>
> As a policy author. you can adapt the metadata.yaml from those provided in the policy template examples. 

> See the following example of a `metadata.yaml`:
> (...)